android/app/build.gradle

```diff
@@ -197,6 +197,7 @@ android {
 }
 
 dependencies {
+    implementation project(':react-native-restart')
```

Comment:
is this needed because some of the i18n stuff is set globally and is somewhat static?

Do you think once we move to react-i18next hooks everywhere, this will go away?

Reply:
The default mobile application for the RTL ( Native behavior ) is restarting the application or change the language from settings ( you can try with facebook for example )
So for example if we don't want to restart we make our views detects the direction and shift every time it changes and there are many approach to do that
**For example In native android I was creating 2 design xml files **rtl** and **ltr** to make android framework handle this by default**  
But here we will do the same thing we did for **typography** to all views

Reply:
I see, if we have to encode the rtl logic into the views anyways, it would be nice if they re-flowed without a restart. 

(later, in a different PR) We could probably make a simple hook:

```js
const dir = useWritingDirection();
```

though, this is not that much different than `const {i18n} = useTranslation()`